README-containers.md

```diff
@@ -191,11 +191,11 @@ All these variables impact in configuration directives in the modsecurity engine
 | ANOMALY_INBOUND | An int indicating the inbound_anomaly_score_threshold (Default: `5`) |
 | ANOMALY_OUTBOUND | An int indicating the outbound_anomaly_score_threshold (Default: `4`) |
 | ALLOWED_METHODS | Allowed_methods (Default: `GET HEAD POST OPTIONS`) |
-| ALLOWED_REQUEST_CONTENT_TYPE | Allowed_request_content_type (Default: `\|application/x-www-form-urlencoded\| \|multipart/form-data\| \|multipart/related\| \|text/xml\| \|application/xml\| \|application/soap+xml\| \|application/x-amf\| \|application/json\| \|application/cloudevents+json\| \|application/cloudevents-batch+json\| \|application/octet-stream\| \|application/csp-report\| \|application/xss-auditor-report\| \|text/plain\|`) |
-| ALLOWED_REQUEST_CONTENT_TYPE_CHARSET | Allowed_request_content_type_charset (Default: `utf-8\|iso-8859-1\|iso-8859-15\|windows-1252`) |
+| ALLOWED_REQUEST_CONTENT_TYPE | Allowed_request_content_type (Default: `\|application/x-www-form-urlencoded\| \|multipart/form-data\| \|multipart/related\| \|text/xml\| \|application/xml\| \|application/soap+xml\| \|application/json\| \|application/cloudevents+json\| \|application/cloudevents-batch+json\|`) |
+| ALLOWED_REQUEST_CONTENT_TYPE_CHARSET | Allowed_request_content_type_charset (Default: `\|utf-8\| \|iso-8859-1\| \|iso-8859-15\| \|windows-1252\|`) |
```

Comment:
@theseion are you sure? Check https://github.com/coreruleset/coreruleset/blob/v3.4/dev/rules/REQUEST-901-INITIALIZATION.conf#L213

Reply:
Weird. I tested extensively and you seem to be right. When I use your proposed syntax, modsecurity blocks as expected, otherwise not. Applying your suggestions

Reply:
Yes, I'm sure :) The link above is for 3.4 but we are targeting 3.3.4: https://github.com/coreruleset/coreruleset/blob/98b9d811f34a1aa72792aaf6245cb2f2c0f0a5b8/rules/REQUEST-901-INITIALIZATION.conf#LL195C36-L195C36